swarm/storage/types.go

```diff
@@ -264,14 +265,32 @@ type Splitter interface {
 	   The caller gets returned an error channel, if an error is encountered during splitting, it is fed to errC error channel.
 	   A closed error signals process completion at which point the key can be considered final if there were no errors.
 	*/
-	Split(io.Reader, int64, chan *Chunk) (Key, func(), error)
+	Split() (Key, func(), error)
 
```

Comment:
Comments of this interface no longer make sense, as they refer to arguments that no longer exist.

Reply:
I realized that we don't even need these interfaces anymore, they are not used anywhere: 424b11dc1b5dd1d0efc46cb9d8a2fe36ebb8e7ef